src/main/java/it/pagopa/pn/deliverypush/dto/timeline/details/NotificationPaidDetails.java

```diff
@@ -17,6 +17,7 @@ public class NotificationPaidDetails implements RecipientRelatedTimelineElementD
     private String noticeCode;
     private String idF24;
     private String paymentSourceChannel;
+    private boolean uncertainPaymentDate;
```

Comment:
Non è relativo a questo sviluppo, cambieresti semplicemente il nome di questa classe da NotificationPaidDetails in NotificationPaidDetailsInt per adeguarla alle restatnti